lib/refile/s3.rb

```diff
@@ -60,7 +59,7 @@ def initialize(region:, bucket:, max_size: nil, prefix: nil, hasher: Refile::Ran
     verify_uploadable def upload(uploadable)
       id = @hasher.hash(uploadable)
 
-      if uploadable.is_a?(Refile::File) and uploadable.backend.is_a?(S3) and uploadable.backend.access_key_id == access_key_id
+      if uploadable.is_a?(Refile::File) and uploadable.backend.is_a?(S3) 
```

Comment:
Hi @seanknox Sorry for the late reply! I don't know if copying between different AWS accounts is an intentional feature but that's how I read the code (comparing access_key_id's, if so - use `#copy_from`). I'll see if I can add stubbed tests. Had done #31 in the meantime (simply didn't know how to make changes to a PR, sorry!).